client/state/selectors/is-fetching-next-page.js

```diff
@@ -2,8 +2,14 @@
  * External dependencies
  */
 import { get } from 'lodash';
+
```

Comment:
Side note: this selector's name is way too broad 😬

Reply:
One parallel effort we should be thinking about is reorganising state.

I seem to recall we had decided that selectors for a particular portion of state should live under `state/{portion}/selectors`, whereas selectors that draw from multiple areas of state should be in `state/selectors`.

In that situation, `state/media/is-fetching-next-page` would be perfectly fine.

Reply:
Agreed. 

I haven't checked thoroughly, but many of the selectors in the so-called "global selectors directory" were put there because it was considered it was going to be the place for ALL selectors at some point in history. As you pointed out, that has changed since, but we haven't really done a thorough audit to move away some of them that are specific to a certain state tree.